backend/src/main/java/gov/cdc/usds/simplereport/idp/authentication/LiveOktaAuthentication.java

```diff
@@ -216,7 +240,7 @@ public String enrollVoiceCallMfa(String userId, String phoneNumber)
       return callFactor.getId();
     } catch (ResourceException e) {
       if (e.getStatus() == HttpStatus.BAD_REQUEST.value()) {
-        throw new BadRequestException(e.getError().getMessage(), e);
+        throw new BadRequestException("Invalid phone number.", e);
```

Comment:
This is the same issue as above, it might be that they're already enrolled in voice call MFA but we don't really have a way of determining that so I'm planning to keep the invalid phone number message.